src/tests/getGoatFacts.test.js

```diff
@@ -0,0 +1,13 @@
+import { getGoatFacts } from '../getGoatFacts'
+
+
+test('Check if the getGoatFacts function exists',() => {
+    expect(getGoatFacts).toBeDefined()
+})
+
+//Note : API endpoint needs to be running for this test to work
+test('Check if async data call to retrieve data is correct', async () => {
```

Comment:
I think this unit tests shouldn't have to depend on the server to be running. especially if we have control over the API endpoint.

what we really want to test is the logic of the function and not so much care about if the connection to the API is successful

is there something else we could do to keep this unit test isolated from that?